Source/Scene/Cesium3DTileGroundPolylines.js

```diff
@@ -0,0 +1,418 @@
+/*global define*/
+define([
```

Comment:
Should we rename this class to something more generic in case it is used for other batched renderers?